skins/cat17/templates/partials/Page_Footer.html.twig

```diff
@@ -17,7 +17,7 @@
 			</div>
 			<nav class="menu menu-footer col-xs-12 col-sm-4 col-md-4">
 				<ul class="list-menu list-unstyled">
-					<li>
```

Comment:
Wouldn't that be ready in `laika` by the time we run the three skins test?

Reply:
I thought the same thing, the requirements dont fully make sense to me to be honest.